doc/thirdpartypackages/index.rst

```diff
@@ -1,179 +1,147 @@
 .. _thirdparty-index:
 
-*********************
- Third party packages
-*********************
+********************
+Third party packages
+********************
 
 Several external packages that extend or build on Matplotlib functionality
-exist. Below we list a number of these. Note that they are each
-maintained and distributed separately from Matplotlib, and will need
-to be installed individually.
+exist. Below we list a number of these. Note that they are each maintained
```

Comment:
I think `Below we list a number of these` is sort of redundant, or at the least should be `We list a number of these below.` Suggested rephrase (and yes, nitpicky)

```
Several external packages that extend or build on Matplotlib functionality exist. They are maintained and distributed separately from matplotlib and so need to be installed individually. We list a number of these below.
```